packages/dds/tree/package.json

```diff
@@ -130,9 +130,11 @@
 		"test:customBenchmarks:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:customBenchmarks",
 		"test:memory": "mocha --perfMode --config ./src/test/memory/.mocharc.cjs",
 		"test:memory-profiling:report": "mocha --config ./src/test/memory/.mocharc.cjs",
-		"test:mocha": "echo skipping non-smoke cjs to avoid overhead && npm run test:mocha:cjs -- --fgrep @Smoke && npm run test:mocha:esm",
+		"test:mocha": "echo skipping non-smoke cjs to avoid overhead && npm run test:mocha:cjs -- --fgrep @Smoke && npm run test:mocha:prod -- --fgrep @Smoke && npm run test:mocha:esm",
 		"test:mocha:cjs": "cross-env FLUID_TEST_MODULE_SYSTEM=CJS mocha",
 		"test:mocha:esm": "mocha",
+		"test:mocha:prod": "mocha --emulateProduction",
+		"test:mocha:prod-help": "echo run tests simulating production configurations as much as possible.",
```

Comment:
Is this a script that runs by this name specifically, or some npm convention to output help for the script in some scenarios? I couldn't find docs about it so I assume the former (so kind of serves the purpose of a comment?).